exec/java-exec/src/main/resources/rest/alertModals.ftl

```diff
@@ -47,26 +47,21 @@
 <script>
     //Populate the alert modal with the right message params and show
     function populateAndShowAlert(errorMsg, inputValues) {
-      //console.log("placeHolder -> "+ JSON.stringify(errorMap[errorMsg].placeHolder));
-      //console.log("inputValues -> "+ JSON.stringify(inputValues));
-      modalHeader.innerHTML=errorMap[errorMsg].msgHeader;
-      modalBody.innerHTML=errorMap[errorMsg].msgBody;
-      let substNames=[];
-      let substValues=[];
+      if (! errorMsg in errorMap) {
```

Comment:
I think this needs to be changed to `if (!(errorMsg in errorMap))`

Reply:
Yes, i missed it. Fixed now